boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -239,6 +251,43 @@ def trim_index(index, kind, item):
     # Get rid of these so we can iterate over what's left at the end
     trim_index(by_kind, kind, 'ALL')
 
+
+csv['spec']['install']['spec']['permissions'] = []
+
+if 'namespace' not in csv['metadata']:
+    raise UndefinedCSVNamespace(OPERATOR_NAME)
+
+# Find namespace of operator from CSV template
+namespace = csv['metadata']['namespace']
+
+if 'Role' in by_kind:
+    filtered_roles = []
+    filtered_rolebindings = []
+    for role in by_kind['Role']:
+        # We assume there is always a rolebinding of defined role
+        if role['metadata']['name'] not in rb_by_role:
+            raise NoAssociatedRoleBinding(role['metadata']['name'], role['metadata']['namespace'])
+        role_binding = rb_by_role[role['metadata']['name']]
+
+        # If the RoleBinding subject is a ServiceAccount in the same namespace as the operator
+        # we add it to CSV. Else it is to be written as-is to the bundle
+        if len(role_binding['subjects']) == 1 and \
+            role_binding['subjects'][0]['kind'] == 'ServiceAccount' and \
+            role_binding['subjects'][0]['namespace'] == namespace:
```

Comment:
/lgtm cancel

Ah, [CI failure](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_boilerplate/141/pull-ci-openshift-boilerplate-master-pr-check/1382373705857896448#1:build-log.txt%3A4013) for a rolebinding with no namespace in the subject. I believe this is a legit configuration -- the namespace defaults to the current one -- so this line could change to something like:
```suggestion
            # If the subject has no namespace, OLM will assume the current one
            role_binding['subjects'][0].get('namespace', namespace) == namespace:
```